docker/sdk_client.go

```diff
@@ -0,0 +1,37 @@
+package docker
+
+import (
+	"errors"
+	"io/ioutil"
+
+	dlib "github.com/fsouza/go-dockerclient"
+)
+
+var (
+	errNotYetImplemented = errors.New("not yet implemented")
+)
+
+type sdkClient struct {
+	cl *dlib.Client
+}
+
+// NewSDKClient creates a new Client that uses the SDK under the hood
+func NewSDKClient(cl *dlib.Client) Client {
+	return &sdkClient{cl: cl}
+}
+
+func (s *sdkClient) Pull(img *Image) error {
```

Comment:
Add documentation strings for these exported funcs?

Or make that a TODO later if this needs to merge now--`deisrel` is essentially an internal tool. 🙈